include/alp/reference/matrix.hpp

```diff
@@ -2251,7 +2251,7 @@ namespace alp {
 
 			template < bool d >
 			struct view_type< view::gather, d > {
-				using type = Matrix< T, structures::General, Density::Dense, view::Gather< self_type >, imf::Strided, imf::Strided, reference >;
+				using type = Matrix< T, structures::Hermitian, Density::Dense, view::Gather< self_type >, imf::Strided, imf::Strided, reference >;
```

Comment:
Not sure we can guarantee this. Perhaps best to keep `General` and once a submatrix is gathered view it as Hermitian if this is acceptable in the particular context where this type cast is needed.

Reply:
This will break `L173` in `householder_tridiag.hpp`
`auto Qk_view = get_view( Qk, utils::range( k + 1, n ), utils::range( k + 1, n ) );`
since we want to use `Qk_view` on the matrix with Symmetric/Hermitian structure in the next `foldl`.